### DISCUSSION TITLE
Completely new here, installations not reopening.

### DISCUSSION THREAD

**Discussion by TremendousTourmaline** [The-Wineskin-Project/WineskinServer#564] [Q&A] [answered]
Greetings everyone! As the title suggests I'm completely new and unfamiliar with Wineskin, and I'm having trouble with some installations. I have a Mac and have been wanting to play Windows games without using a Bootcamp (I don't want to split my SSD), or with another software forcing me to pay 1 and a quarter grand every year. Being a complete newbie, I'm obviously having a hard time, and decided to ask for help.

For context, I'm on a **MacOS Sonoma 14.2.1**, using the latest version of Wineskin Winery as well as Homebrew.

<img width="276" alt="Screenshot 2024-04-02 at 9 04 25 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/f278cb67-4c4e-455a-8c53-81473b1db05a">

<img width="459" alt="Screenshot 2024-04-02 at 9 06 20 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/e323e986-53ee-4246-9593-b7d053a67f33">

So in my case, to start with, I'm trying to install a Windows game called "Mechanism," seen here,

<img width="228" alt="Screenshot 2024-04-02 at 9 10 27 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/63716ad2-1eff-49e2-920e-8c102ceda2fc">

In the winery, I've installed several new engines, and unfortunately I don't really know the difference between them.

<img width="402" alt="Screenshot 2024-04-02 at 9 09 14 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/ef026989-00a6-4e86-8bcd-b04635d8050c">

I created a wrapper called "Test Wrapper" and it worked fine.

<img width="368" alt="Screenshot 2024-04-02 at 9 10 02 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/f6617068-2042-4ef5-82b0-c465610169fc">

So I then choose setup executable and then the file of my choice, in this case, mechanism.exe.

<img width="599" alt="Screenshot 2024-04-02 at 9 10 14 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/6d1771fe-bcae-4e26-8838-7e7e313d8860">

And... the game works perfectly!

<img width="495" alt="Screenshot 2024-04-02 at 9 10 43 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/e5b98f67-2065-4eeb-a130-55026ce0cb89">

<img width="1440" alt="Screenshot 2024-04-02 at 9 11 28 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/6a6fb9c1-5cb0-4c28-b67d-d7916edc3b4e">

The name of the installation appears as so:

<img width="187" alt="Screenshot 2024-04-02 at 9 14 31 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/fab25dd1-9956-4b30-89dc-30e0385b1c28">

When I ask to show the installation at the finder, here's what shows up.

<img width="920" alt="Screenshot 2024-04-02 at 9 14 50 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/12f17667-6139-45d9-9db2-eeaa3700c9c4">

However, whenever closing the game, the installation completely disappears. If I asked the game to create a shorcut in the desktop folder, after I quit, a file called "Mechanism.ink" appears in its place:

<img width="100" alt="Screenshot 2024-04-02 at 9 32 48 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/0928c44a-4af3-4b51-9e11-9cc01f20ccab">

When I went into the aforementioned "show in finder" and clicking on "wine-64 preloader" which was the name of the app on the sidebar, the Terminal opens up and this code runs. Then nothing happens, or at least I haven't noticed anything happen.

<img width="568" alt="Screenshot 2024-04-02 at 9 17 40 AM" src="https://github.com/Gcenx/WineskinServer/assets/126363873/1330ae6f-7424-4b82-8fe2-015bc2718425">

Finally, and I unfortunately closed this before taking the screenshot and I'm too tired to do the whole process again (I've installed and uninstalled/deleted this game and repeated the procedure four times at this point), but after the Terminal finished, Wineskin told me that they couldn't find... the installation source or something?

All and all, the installation part works fine, but when I quit and try to reopen a Windows app, it won't work.
The answer is probably obvious and easy to solve, but being new, I have no idea, so I appreciate any help.
I have already searched for a similar discussion and have not found anyone bring up this issue. Apologies if there is one and I missed it.

If anyone would like to try to replicate, the game's download page is here: https://gamejolt.com/games/Mechanism/811356

**Comment by Gcenx** [comment]
You need to remember _where_ you’ve installed this game to.

If it’s installed into your user directory Wineskin won’t find it, you need to install things into Program Files for Wineskin to find it.

**Reply by TremendousTourmaline** [reply]
I was able to finally find Wineskin's notice that they can't find the executables, which I'm assuming has to do with what you mentioned, that I didn't install in the proper place:

![image](https://github.com/Gcenx/WineskinServer/assets/126363873/babab7db-e3bb-41a9-9669-ff0585f83dbc)

> install things into Program Files for Wineskin to find it.

Could you elaborate on this? Which folder in drive_c --> programmer files should I put in into, if any? Should I just stick it into the programmer files folder?

**Reply by emendelson** [reply, marked as answer]
<img width="215" alt="Screenshot 2024-04-03 at 1 50 14 PM" src="https://github.com/Gcenx/WineskinServer/assets/5208505/83ac6031-8e01-4ca1-9e8c-48e338e2e435">

**Reply by TremendousTourmaline** [reply]
Thank you!
